src/D2L.CodeStyle.Analyzers/Immutability/ImmutabilityContext.cs

```diff
@@ -112,7 +112,7 @@ out diagnostic
 					diagnostic = Diagnostic.Create(
 						Diagnostics.ArraysAreMutable,
 						location,
-						type.Name
+						( type as IArrayTypeSymbol ).ElementType.Name

```

Comment:
Rebase?

Reply:
Done. Had opened from my phone. Was hoping it just wouldn't show.

Reply:
Yeah, I wish GitHub hadn't switched how they calculated diffs :(